webrender/src/glyph_cache.rs

```diff
@@ -89,6 +90,14 @@ impl GlyphCache {
         }
     }
 
+    pub fn clear(&mut self, texture_cache: &mut TextureCache) {
+        let caches = mem::replace(&mut self.glyph_key_caches, FastHashMap::default());
```

Comment:
this could be rewritten cleaner without `mem::replace`, which I consider highly overused in our codebase, e.g.:
```rust
for (_, mut glyph_key_cache) in &mut self.glyph_key_caches {
  glyph_key_cache.clear(texture_cache);
}
self.glyph_key_caches.clear();
```
Bonus points - space is kept reserved.